common/persistence/sql/plugins/mysql/events.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
leave the copyright year as is since this file contains changes from 2017-till now

Reply:
Got it.